packages/api/src/schema/compile-schemas.js

```diff
@@ -24,23 +24,47 @@ const write = (dir, data) => {
   console.log(`wrote ${dir}`);
 };
 
-const schemaDir = path.resolve(__dirname, "schema");
+// Remove the title from the schema to avoid conflicts with the TypeScript type name
+function removeAllTitles(schema) {
```

Comment:
Just a thought, any risk of this affecting generating tools like documentation tools or sdk generating tools, or they are going to be merged anyway? If any of them require `title`

Reply:
This will not affect SDK/docs, since it's only used in build time to generate that combined schema that we use in our code! The SDK/docs will use the raw `api-schema.yaml` and `ai-api-schema.yaml` fields.